YesterdayApi/Core/Identity/IdentityService.cs

```diff
@@ -0,0 +1,125 @@
+using System.Net.Http;
+using System.Threading.Tasks;
+using AutoMapper;
+using IdentityModel.Client;
+using IdentityServer.Core.Users;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
+using YesterdayApi.Core.Identity.Web;
+using YesterdayApi.Core.Users;
+using YesterdayApi.Utilities.Exceptions;
+
+namespace YesterdayApi.Core.Identity
+{
+    public class IdentityService : IIdentityService
+    {
+        private const string DiscoveryUrl = "http://localhost:5000";
```

Comment:
Maybe would be better to put this url into the configuration like connection string?

Reply:
Idk, i think it's specific to this one class, so it kinda makes sense to keep it cohesive.

Reply:
Yeah nvm, found your second comment on this string down the line, nice catch.